interp/tests/add_feeding.expect

```diff
@@ -5,36 +5,36 @@
         "out": 1
       },
       "add1": {
-        "left": 1,
-        "out": 2,
-        "right": 1
+        "left": 0,
+        "out": 0,
+        "right": 0
```

Comment:
Why did everything become zeroes in the tests?

Reply:
We talked about this through another channel, but just to put this here for everyone's benefit. Rather than printing at the moment before a group's execution finishes, we are now printing after a group finishes executing (i.e. the go signal to the group shuts off and all the assignments within cease, leaving only valid values on some output ports, such as reg.out).

Reply:
This is consistent with the behavior of the other multi-group tests, though points out that we should think about ways to make the simulation action of the interpreter more interrogable (perhaps via attributes) and consider what information would be useful to emit